.gitignore

```diff
@@ -1,26 +1,28 @@
-zingocli/regtest/data/zcashd/regtest/
+!zingolib/test-data/README
+!zingolib/test-data/openssl_cfg
+*.json.txt
+.DS_Store
+/procgov
+/testdata
+cobertura.xml
+darkside-tests/lightwalletd_bin/
```

Comment:
should this be removed as no longer used (old symlinks not being removed automatically)?

Reply:
yes